app/controllers/hubstats/deploys_controller.rb

```diff
@@ -27,6 +27,14 @@ def index
     end
 
     def show
+      @deploy = Hubstats::Deploy.find(params[:id])
+      @repo = @deploy.repo
+      @pull_requests = Hubstats::PullRequest.belonging_to_deploy(@deploy.id)
+      @pull_request_count = Hubstats::PullRequest.belonging_to_deploy(@deploy.id)
```

Comment:
Don't go back to the database when you already have the object you need data from.  Also, do you mean to call size on this? `pull_request_count = @pull_requests.size`